lib/three_scale/middleware/multitenant.rb

```diff
@@ -33,7 +33,7 @@ def initialize(attribute, app, env)
         def verify!(object)
 
           # this controller shouldnt be checked
-          return true if @env["action_controller.request.path_parameters"]["controller"] == "provider/domains"
+          return true if @env["action_dispatch.request.path_parameters"]["controller"] == "provider/domains"
```

Comment:
seems like this has been the case since Rails 3.0

https://apidock.com/rails/v3.0.0/ActionDispatch/Http/Parameters/path_parameters
https://apidock.com/rails/v2.3.8/ActionController/Request/path_parameters

Reply:
This seems hacky, like messing with Rails internals, so it can be broken at every Rails update we make. If we are sure there will be a controller at this point, isn't there any other standard way to get it?